web/pom.xml

```diff
@@ -66,6 +66,11 @@
             <artifactId>graphhopper-web-bundle</artifactId>
             <version>1.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
```

Comment:
I was a little surprised I had to add this, as I expected this module would have access to the `web-bundle` `8.0-SNAPSHOT` via transitive dependencies, but that didn't work. Adding the dep explicitly here seemed to fix the issue 🤷